src/services/codefixes/fixUnusedIdentifier.ts

```diff
@@ -154,6 +154,13 @@ namespace ts.codefix {
         }
         if (isIdentifier(token) && canPrefix(token)) {
             changes.replaceNode(sourceFile, token, createIdentifier(`_${token.text}`));
+            if (isParameter(token.parent)) {
+                getJSDocParameterTags(token.parent).forEach((tag) => {

```

Comment:
Don’t you need to do this only for the JSDoc parameter tags whose name is the same as `token.text`? Try adding this test:

```ts
/**
 * @param a
 * @param b
 */
function f(a, b) {
    const x = a;
}
```

to ensure that `b` gets renamed but `a` does not.

Reply:
Ah, I see `getJSDocParameterTags` already handles that. My bad. Thanks for adding the test anyway.